src/slurm_plugin/clustermgtd.py

```diff
@@ -247,6 +248,9 @@ def _get_terminate_config(self, config):
         self.protected_failure_count = config.getint(
             "clustermgtd", "protected_failure_count", fallback=self.DEFAULTS.get("protected_failure_count")
         )
+        self.insufficient_capacity_timeout = config.getfloat(
+            "clustermgtd", "insufficient_capacity_timeout", fallback=self.DEFAULTS.get("insufficient_capacity_timeout")
+        )
```

Comment:
we could also set an additional field self.disable_nodes_on_insufficient_capacity = self.insufficient_capacity_timeout > 0 and drop _is_smart_instance_capacity_fail_over_enabled

Reply:
Changed to `self.disable_nodes_on_insufficient_capacity = self._config.insufficient_capacity_timeout > 0`